mmv1/third_party/terraform/website/docs/d/apphub_discovered_workload.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Apphub"
+description: |-
+  Get information about a discovered workload.
+---
+
+# google\_apphub\_discovered_workload
+
+Get information about a discovered workload from its uri.
+
+
+## Example Usage
+
+
+```hcl
+data "google_apphub_discovered_workload" "my-workload" {
+  location = "my-location"
+  workload_uri = "my-workload-uri"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `workload_uri` - The uri of the workload.
+* `location` - The location you want to get the discovered worklaod from
```

Comment:
Can you please add `(Required) ` to these two lines?
Example
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/d/access_approval_folder_service_account.html.markdown?plain=1#L35C17-L35C28

Reply:
Done.